docs/shared/recipes/plugins/create-install-package.md

```diff
@@ -0,0 +1,159 @@
+# Creating an Install Package
+
+{% youtube
+src="https://www.youtube.com/embed/ocllb5KEXZk"
+title="Build your own CLI"
+width="100%" /%}
+
+A [Nx Plugin Preset](/extending-nx/recipes/create-preset) allows you to take control over the shape of an Nx workspace setup, so you can customize it to best fit your situation. This is possible because a preset can be passed directly to the `create-nx-workspace` command:
```

Comment:
```suggestion
An [Nx Plugin Preset](/extending-nx/recipes/create-preset) allows you to take control over the shape of an Nx workspace setup, so you can customize it to best fit your situation. This is possible because a preset can be passed directly to the `create-nx-workspace` command:
```